legend-sdlc-server/src/main/resources/docker/config/config.json

```diff
@@ -0,0 +1,106 @@
+{
+  "applicationName": "Legend SDLC",
+  "server": {
+    "adminContextPath": "/admin",
+    "applicationConnectors": [
+      {
+        "type": "http",
+        "port": ${SDLC_PORT},
+        "maxRequestHeaderSize": "128KiB"
+      }
+    ],
+    "adminConnectors": [
+      {
+        "type": "http",
+        "port": 7071
+      }
+    ],
+    "gzip": {
+      "includedMethods": [
+        "GET",
+        "POST"
+      ]
+    },
+    "requestLog": {
+      "type": "classic",
+      "level": "OFF",
+      "appenders": [
+        {
+          "type": "console",
+          "logFormat": "OFF"
+        }
+      ]
+    },
+    "rootPath": "/api"
+  },
+  "projectStructure": {
+    "projectCreation": {
+      "groupIdPattern": "^org\\.finos\\.legend\\..+"
+    },
+    "extensionProvider": {
+      "org.finos.legend.sdlc.server.gitlab.finos.FinosGitlabProjectStructureExtensionProvider": {}
+    }
+  },
+  "filterPriorities": {
+    "GitLab": 1,
+    "org.pac4j.j2e.filter.CallbackFilter": 2,
+    "org.pac4j.j2e.filter.SecurityFilter": 3,
+    "CORS": 4
+  },
+  "pac4j": {
+    "callbackPrefix": "/api/pac4j/login",
+    "mongoUri": "${MONGODB_URI}",
+    "mongoDb": "${MONGODB_NAME}",
+    "clients": [
+      {
+        "org.finos.legend.server.pac4j.gitlab.GitlabClient": {
+          "name": "gitlab",
+          "clientId": "${GITLAB_APP_ID}",
+          "secret": "${GITLAB_APP_SECRET}",
+          "discoveryUri": "https://${GITLAB_HOST}/.well-known/openid-configuration",
+          "scope": "openid profile api"
+        }
+      }
+    ],
+    "mongoSession": {
+      "enabled": "${MONGO_SESSION_ENABLED}",
+      "collection": "userSessions"
+    },
+    "bypassPaths": [
+      "/api/info"
+    ]
+  },
+  "features": {
+    "canCreateProject": true,
+    "canCreateVersion": true
+  },
+  "gitLab": {
+    "newProjectVisibility": "public",
+    "projectTag": "pure",
+    "projectIdPrefix": "UAT",
+    "server": {
+      "scheme": "https",
+      "host": "${GITLAB_HOST}"
+    },
+    "app": {
+      "id": "${GITLAB_APP_ID}",
+      "secret": "${GITLAB_APP_SECRET}",
+      "redirectURI": "http://localhost:7070/api/auth/callback"
```

Comment:
This URL suggest this is for casual users just trying things out on their local computers. I have trouble seeing another group of users for whom this would be an appropriate redirectURI. Should this perhaps be a variable too?

Also, I note the port is hard coded here, whereas it is a variable earlier in the config.

Reply:
Good catch again. I will fix